Gemfile

```diff
@@ -2,11 +2,11 @@
 
 source "https://rubygems.org"
 
-ruby ">= 3.1.0"
+ruby ">= 3.0.0"
```

Comment:
Why this change?

Reply:
@jemmaissroff incorrectly committed on this branch. I reverted